### DISCUSSION TITLE
Still says "Loading" in the top bar and 0 countries in the sidebar

### DISCUSSION THREAD

**Discussion by BenyAlbatross** [deckerst/aves#980] [Q&A]
I've downloaded Aves for a while, but it still says Loading under collections. I assume this means that the gallery is still building a cache? I also noticed that in the sidebar under countries, it says 0, even though it is able to identify the country based on the GPS coordinates in individual pictures. 

In the stats menu, it also shows 1 item with location and the top country has only 1 photo, even though there are many photos in this country.

How can I solve this?

**Comment by deckerst** [comment]
After you kill the app and relaunch it, does it start on "loading" forever? 
How many items do you have in your collection, more or less? 10k? 100k?

Also, please provide the logs and system info you get from About > Bug Report.

**Reply by BenyAlbatross** [reply]
I cleared some storage on my phone (~2gb free) and killed the app and relaunched it, it works now. I think it lacked space to create the cache. Collection is about 30k items.  Just in case this is unintended behaviour or you are considering adding a warning for users with insufficient storage, here are the logs and system info:

System info:
Package: deckers.thibault.aves
Installer: com.android.vending
Aves version: 1.10.8-play, build 117
Flutter: stable 3.19.5
Android version: 14, API 34
Android build: AP1A.240405.002
Device: Google Pixel 7
Support: dynamic colors=true, geocoder=true, HDR=true
Mobile services: ready
System locales: en_US, en_GB
Storage volumes: /storage/emulated/0/
Storage grants: 
Error reporting: false
[aves-logs-20240418_005812.txt](https://github.com/deckerst/aves/files/15014854/aves-logs-20240418_005812.txt)

**Reply by deckerst** [reply]
Thanks for letting me know.

About collection size, I see in your logs
```
04-18 00:56:42.751  1976  2096 I flutter : Report log=Source refresh complete in 3s for 53541 known, 19 new, 4 removed
```
In other words, 50k items loaded in 3 seconds. On my device the DB for ~15k items takes 5 MB, so for you it might be ~20 MB. You could check in `About` > `Data Usage`. In any case, you probably had enough storage for the DB so there must be something else going on, and I don't see any error in the logs.
